somecode.cpp

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
[Problematic C function detected (memcpy)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS121708.md#DS121708) 7 times
There are a number of conditions in which memcpy can introduce a vulnerability (mismatched buffer sizes, null pointers, etc.). More secure alternitives perform additional validation of the source and destination buffer

*Fix Guidance:* Use memcpy_s if possible.


Suggestions:
[1]